packages/skill-api/src/core/services/process-sanity-webhooks.ts

```diff
@@ -31,23 +30,23 @@ export async function processSanityProductsWebhook({
           body: 'Invalid webhook signature',
         }
       }
-      const product = req.body
-      console.log({product, req, params})
-      if (process.env.INNGEST_EVENT_KEY && product) {
+      const data = req.body
+
+      if (process.env.INNGEST_EVENT_KEY && data) {
         const inngest = new Inngest({
           id:
             process.env.INNGEST_APP_NAME ||
             process.env.NEXT_PUBLIC_SITE_TITLE ||
             'Sanity Products Webhook',
           eventKey: process.env.INNGEST_EVENT_KEY,
         })
-        console.log('sending inngest event')
-        await inngest.send({
-          name: SYNC_SANITY_PRODUCT,
-          data: {
-            product: req.body,
-          },
-        })
+        console.log('sending inngest event', data)
+        if (data._type === 'product') {
+          await inngest.send({
+            name: SYNC_SANITY_PRODUCT,
+            data,
+          })
+        }
```

Comment:
this is where we can choose which event to send based on `_type` or `event` coming through the webhook